src/CoreShop/Component/Variant/Event/variantAvailabilityEvent.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * CoreShop
+ *
+ * This source file is available under two different licenses:
+ *  - GNU General Public License version 3 (GPLv3)
+ *  - CoreShop Commercial License (CCL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ * @copyright  Copyright (c) CoreShop GmbH (https://www.coreshop.org)
+ * @license    https://www.coreshop.org/license     GPLv3 and CCL
+ *
+ */
+
+namespace CoreShop\Component\Variant\Event;
+
+use CoreShop\Component\Core\Model\ProductInterface;
+use Symfony\Contracts\EventDispatcher\Event;
+
+class variantAvailabilityEvent extends Event
```

Comment:
```suggestion
class VariantAvailabilityEvent extends Event
```

The file also needs to be renamed.